addons/event_stand/models/event_stand.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventStand(models.Model):
```

Comment:
Other question: don't we need activities on it ? Maybe not, but could be useful if you have some menus, for example "call customer to validate registration XYZ". Maybe also to check with LNA.